docs/process/style_guide.md

```diff
@@ -938,4 +938,3 @@ Thus we could provide one or more ICU4X traits bound to things like `str` to pro
   * You can save snippets in permantent links and incluce them as working examples in docs.
 * Write code and see what it compiles to: https://rust.godbolt.org
   * Note that you need to enable compiler optimizations via `-C opt-level=3` if you are looking to meaningfully compare two code snippets.
-
```

Comment:
There were double newlines at the end of this file, and my editor automatically changed it to one.